src/components/HotelPageFetch/HotelPageFetch.js

```diff
@@ -1,10 +1,14 @@
 import {useParams} from "react-router-dom";
-import {useEffect, useState} from "react";
+import {useEffect, useState,} from "react";
 
 const HotelPageFetch = () => {
     const {id} = useParams()
     const url = `https://fe-student-api.herokuapp.com/api/hotels/${id}`;
```

Comment:
why do you store static path here in the component?

Reply:
did this